caffe2/sgd/learning_rate_op.cc

```diff
@@ -163,8 +163,10 @@ C10_EXPORT_CAFFE2_OP_TO_C10_CPU(
     "int? inactive_period = -1, "
     "int? max_iter = -1, "
     "int? num_iter = 0, "
+    "float? decay = 1.0, "
```

Comment:
please add new parameter at the end, and make sure the default value matches the original behavior.